app/scripts/templates/complete_2fa.mustache

```diff
@@ -0,0 +1,18 @@
+<div id="main-content" class="card">
+  <header>
+    <h1 id="fxa-confirm-header">{{#t}}New Sign-in to Firefox{{/t}}</h1>
```

Comment:
Two `h1`s is a bit strange, usually only one is used on a page. Is the 2nd one so that the text is bold also?

The IDs used on these two elements are used in other pages, can you create new IDs so that we can easily differentiate this page from other pages in our functional tests? Maybe `fxa-2fa-confirm-signin-header`